.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -1,18 +1,20 @@
 channels:
   - pytorch
   - defaults
+  # using conda-forge for python v3.9+
+  - conda-forge
 dependencies:
-  - numpy
+  - conda-forge::numpy
   - pytest
   - pytest-cov
   - codecov
   - pip
   - libpng
-  - jpeg
-  - ffmpeg=4.2
+  - jpeg <= 9b
+  - ffmpeg >=4.2
```

Comment:
cc @andfoy, `ffmpeg =4.2` wasn't actually working with `python 3.9+` we may need to rebuild the packages for `pytorch/builder`.

Until then I'm allowing the packages to use the `ffmpeg` from `conda-forge`

Reply:
Thanks for the notice! I'll rebuild the packages accordingly